indigo_app/templates/place/works.html

```diff
@@ -153,6 +157,7 @@
                             </div>
                           {% endif %}
                         </div>
+  </div>
```

Comment:
this is breaking things

Reply:
you need to move the pagination inside the second `.col`

Reply:
Fixed :white_check_mark: